README.md

```diff
@@ -1,10 +1,29 @@
 # Variation Normalization
-Services and guidelines for normalizing variation terms
+Services and guidelines for normalizing variation terms into [VRS](https://vrs.ga4gh.org/en/latest/) and [VRSATILE](https://vrsatile.readthedocs.io/en/latest/) compatible representations.
+
+FastAPI: https://normalize.cancervariants.org/variant
+
+## About
+Variation Normalization works by using four main steps: tokenization, classification, validation, and translation. During tokenization, we split strings on whitespace and parse to determine the type of token. During classification, we specify the order of tokens a classification can have. We then do validation checks such as ensuring references for a nucleotide or amino acid matches the expected value and validating a position exists on the given transcript. During translation, we return a VRS Allele object.
+
+Variation Normalization is limited to substitution, deletion, insertion, and deletion-insertion variants located on p., c., and g. coordinates. We also support HGVS representations and text representation (ex: `BRAF V600E`). We are working towards adding more types of variants, coordinates, and representations.
+
+### Endpoints
+#### /toVRS
+The `/toVRS` endpoint returns a list of valid [Alleles](https://normalize.cancervariants.org/variant). 
+
+#### /normalize
+The `/normalize` endpoint returns a [Variation Descriptor](https://normalize.cancervariants.org/variant) containing the MANE Transcript, if one is found.
```

Comment:
Would it be helpful to provide links to the VRS specs for these objects as well?

Reply:
Hahaha I meant to put the respective links for Allele and Variation Descriptor. I'm not sure what happened with my copying and pasting